frontend/src/components/InboxRequest/index.js

```diff
@@ -1,8 +1,9 @@
 import React from "react";
-import { List, message, Image } from "antd";
+// import { List, message, Image } from "antd";
+import { List, message } from "antd";
 import { getRequest } from "../../requests/requestFriendRequest";
-import ReactMarkdown from "react-markdown";
-import PostDisplay from "../PostDisplay";
+// import ReactMarkdown from "react-markdown";
+// import PostDisplay from "../PostDisplay";
```

Comment:
If these few lines of code are useless, should they be deleted?

Reply:
The only reason they are still there is because I neither know who is leaving them there nor what is the purpose, so I just comment them out that the owner of that snippet will be easy to restore, or even better, they are no longer needed, then we can clean the code up without any worry...